core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4329,7 +4331,8 @@ components:
           description: The type of the entity
         attributes:
           type: object
-          properties: {}
+          additionalProperties:
+            type: string
```

Comment:
I tried using `Attribute` but the json content can't be coerced to that type. Looking the json, it was only a `string`. I can update the `description` too if that makes sense. Or remove the descriptions since they're now redundant.

Reply:
This could be a string, a boolean, a number, an array of any of the previous, or an embedded json object - it's not just a string.

Reply:
OK, I wasn't sure if the results included array types.  Should I PR a fix that changes this to `type: object`, so it ends up as `Map<String, Object>` in Java?